ha-jobs-core/src/main/scala/de/kaufhof/hajobs/JobStatus.scala

```diff
@@ -89,6 +89,8 @@ trait JobTypes {
     case unknown => throw new NoSuchElementException(s"Could not find JobType with name '$unknown'.")
   }
 
+  def getAll: List[JobType]
```

Comment:
- If there's a list of all job types, then the byName PF should be removed.
- Naming: I'd prefer just "all" instead of "getAll" (getAll is more java'ish).